core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCardList.kt

```diff
@@ -37,7 +38,7 @@ import com.google.samples.apps.nowinandroid.core.model.data.NewsResource
  */
 fun <T> LazyListScope.newsResourceCardItems(
     items: List<T>,
-    newsResourceMapper: (item: T) -> NewsResource,
+    newsResourceMapper: (item: T) -> UserNewsResource, // TODO remove this?
```

Comment:
We should address this before merging

Reply:
@tunjid Can't wait to have you back on this project 😄 ... 

Why was this function implemented as a generic function with a mapper? Presumably this was to allow `item` to be mapped to a `NewsResource` regardless of its type (via `newsResourceMapper`), but I'm not sure how this was ever applicable. For now, I'm going to remove the generic type to simplify this function but wanted to leave a note here so we can discuss at some future date.